### ISSUE TITLE
A problem about LSTM learning

### ISSUE THREAD

**Issue by LinRick** [maxpumperla/elephas#141]
Dear sir,
When I use the following LSTM architecture along with keras/elephas, as:
conf = SparkConf().setAppName('RNN_LSTM_Elephas_App').setMaster('spark://master:7077')
# one driver and four workers
sc = SparkContext(conf=conf) 

rdd = to_simple_rdd(sc, train_X, train_y)    

model = Sequential()
model.add(LSTM(32, input_shape=(train_X.shape[1],train_X.shape[2]), activation='tanh', kernel_initializer='lecun_uniform'))
model.add(Dense(1))
model.compile(loss="mse", optimizer='adam')
sparkRNN_model = SparkModel(model, frequency='epoch', mode='asynchronous') 
sparkRNN_model.fit(rdd, epochs = 150, batch_size = 32, verbose = 2, validation_split = 0.1)

The predicting results are the same values as follows:
array([[33.390114],
       [33.390114],
       [33.390114],
       [33.390114],
       ..., [33.390114]]

By the way, when I use setMaster('local[*]'), the results are different values.
Hence, I have no idea about this problem.

Rick

**Comment by danielenricocahall** [comment]
Elephas currently doesn't support LSTM training/prediction - it can be added if enough users request it! Although I would definitely want some assistance/feedback.

**Comment by danielenricocahall** [comment]
Moved this issue to the new fork: https://github.com/danielenricocahall/elephas/issues/10. Closing this for now but still on the radar!
